Module not found: Error: Can't resolve '../../constants/disciplines' in '/src/app/pages/lab'

#3860 introduced a build failure which is stopping deploys from working (unfortunately the colour codes make this hard to read):

```
[1m[33mWARNING in webpack: Using NoErrorsPlugin is deprecated.
Use NoEmitOnErrorsPlugin instead.
[39m[22m

[1m[31mERROR in ./app/pages/lab/project-details.cjsx
Module not found: Error: Can't resolve '../../constants/disciplines' in '/src/app/pages/lab'
 @ ./app/pages/lab/project-details.cjsx 3:19-57
 @ ./app/router.cjsx
 @ ./app/main.cjsx
 @ multi ./app/main.cjsx[39m[22m

[1m[31mERROR in ./app/components/filmstrip.jsx
Module not found: Error: Can't resolve '../constants/disciplines' in '/src/app/components'
 @ ./app/components/filmstrip.jsx 13:19-54
 @ ./app/pages/projects/project-filtering-interface.jsx
 @ ./app/pages/projects/filtered-projects-list.jsx
 @ ./app/router.cjsx
 @ ./app/main.cjsx
 @ multi ./app/main.cjsx[39m[22m
Child [1mhtml-webpack-plugin for "index.html"[39m[22m:
       [0] [1m./~/lodash/lodash.js[39m[22m 540 kB {[1m[33m0[39m[22m}[1m[32m [built][39m[22m
           [] -> factory:[1m[31m1398ms[39m[22m building:[1m[31m3650ms[39m[22m = [1m[31m5048ms[39m[22m
       [1] [1m./~/html-webpack-plugin/lib/loader.js!./views/index.ejs[39m[22m 4.5 kB {[1m[33m0[39m[22m}[1m[32m [built][39m[22m
            factory:[1m144ms[39m[22m building:[1m149ms[39m[22m = [1m[32m293ms[39m[22m
       [2] [1m(webpack)/buildin/global.js[39m[22m 509 bytes {[1m[33m0[39m[22m}[1m[32m [built][39m[22m
           [] -> factory:[1m[31m6301ms[39m[22m building:[1m[31m49480ms[39m[22m = [1m[31m55781ms[39m[22m
       [3] [1m(webpack)/buildin/module.js[39m[22m 517 bytes {[1m[33m0[39m[22m}[1m[32m [built][39m[22m
           [] -> factory:[1m[31m6302ms[39m[22m building:[1m[31m49480ms[39m[22m = [1m[31m55782ms[39m[22m
Child [1mextract-text-webpack-plugin[39m[22m:
       [0] [1m./~/css-loader/lib/css-base.js[39m[22m 2.26 kB {[1m[33m0[39m[22m}[1m[32m [built][39m[22m
           [] -> factory:[1m[31m1288ms[39m[22m building:[1m[31m2569ms[39m[22m = [1m[31m3857ms[39m[22m
       [1] [1m./~/css-loader?{"root":"../public"}!./~/chartist/dist/chartist.min.css[39m[22m 11.1 kB {[1m[33m0[39m[22m}[1m[32m [built][39m[22m
           [] -> factory:9ms building:[1m[33m637ms[39m[22m dependencies:[1m[33m642ms[39m[22m = [1m[31m1288ms[39m[22m
       [2] [1m./~/css-loader?{"root":"../public"}!./~/drag-reorderable/style.css[39m[22m 267 bytes {[1m[33m0[39m[22m}[1m[32m [built][39m[22m
           [] -> factory:8ms building:[1m[33m637ms[39m[22m dependencies:[1m[33m643ms[39m[22m = [1m[31m1288ms[39m[22m
       [3] [1m./~/css-loader?{"root":"../public"}!./~/rc-slider/assets/index.css[39m[22m 6.56 kB {[1m[33m0[39m[22m}[1m[32m [built][39m[22m
           [] -> factory:9ms building:[1m[33m638ms[39m[22m dependencies:[1m[33m641ms[39m[22m = [1m[31m1288ms[39m[22m
       [4] [1m./~/css-loader?{"root":"../public"}!./~/react-select/dist/react-select.css[39m[22m 9.45 kB {[1m[33m0[39m[22m}[1m[32m [built][39m[22m
           [] -> factory:9ms building:[1m[33m639ms[39m[22m dependencies:[1m[33m640ms[39m[22m = [1m[31m1288ms[39m[22m
       [5] [1m./public/assets/builder-landing.jpg[39m[22m 82 bytes {[1m[33m0[39m[22m}[1m[32m [built][39m[22m
           [] -> factory:[1m[31m5432ms[39m[22m building:[1m[31m1014ms[39m[22m = [1m[31m6446ms[39m[22m
       [6] [1m./public/assets/simple-pattern.png[39m[22m 82 bytes {[1m[33m0[39m[22m}[1m[32m [built][39m[22m
           [] -> factory:[1m[31m5341ms[39m[22m building:[1m[31m1115ms[39m[22m = [1m[31m6456ms[39m[22m
       [7] [1m./~/css-loader?{"root":"../public"}!./~/stylus-loader!./css/main.styl[39m[22m 281 kB {[1m[33m0[39m[22m}[1m[32m [built][39m[22m
            factory:96ms building:[1m[31m246313ms[39m[22m = [1m[31m246409ms[39m[22m

[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Linux 3.13.0-123-generic
[0m[37;40mnpm[0m [0m[31;40mERR![0m [0m[35margv[0m "/usr/bin/nodejs" "/usr/bin/npm" "run" "_build"
[0m[37;40mnpm[0m [0m[31;40mERR![0m [0m[35mnode[0m v6.11.0
[0m[37;40mnpm[0m [0m[31;40mERR![0m [0m[35mnpm [0m v3.10.10
[0m[37;40mnpm[0m [0m[31;40mERR![0m [0m[35mcode[0m ELIFECYCLE
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m panoptes-front-end@ _build: `export HEAD_COMMIT=$(git rev-parse --short HEAD); check-engines && check-dependencies && rimraf dist; webpack --config ./webpack.build.js --progress --profile --colors`
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Exit status 2
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m 
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Failed at the panoptes-front-end@ _build script 'export HEAD_COMMIT=$(git rev-parse --short HEAD); check-engines && check-dependencies && rimraf dist; webpack --config ./webpack.build.js --progress --profile --colors'.
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Make sure you have the latest version of node.js and npm installed.
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m If you do, this is most likely a problem with the panoptes-front-end package,
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m not with npm itself.
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Tell the author that this fails on your system:
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m     export HEAD_COMMIT=$(git rev-parse --short HEAD); check-engines && check-dependencies && rimraf dist; webpack --config ./webpack.build.js --progress --profile --colors
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m You can get information on how to open an issue for this project with:
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m     npm bugs panoptes-front-end
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Or if that isn't available, you can get their info via:
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m     npm owner ls panoptes-front-end
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m There is likely additional logging output above.
[0m
[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Please include the following file with any support request:
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m     /src/npm-debug.log
[0m
[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Linux 3.13.0-123-generic
[0m[37;40mnpm[0m [0m[31;40mERR![0m [0m[35margv[0m "/usr/bin/nodejs" "/usr/bin/npm" "run" "_stage"
[0m[37;40mnpm[0m [0m[31;40mERR![0m [0m[35mnode[0m v6.11.0
[0m[37;40mnpm[0m [0m[31;40mERR![0m [0m[35mnpm [0m v3.10.10
[0m[37;40mnpm[0m [0m[31;40mERR![0m [0m[35mcode[0m ELIFECYCLE
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m panoptes-front-end@ _stage: `export PATH_ROOT=preview.zooniverse.org/panoptes-front-end; npm run _build && npm run _publish && npm run _log-staging-url`
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Exit status 1
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m 
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Failed at the panoptes-front-end@ _stage script 'export PATH_ROOT=preview.zooniverse.org/panoptes-front-end; npm run _build && npm run _publish && npm run _log-staging-url'.
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Make sure you have the latest version of node.js and npm installed.
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m If you do, this is most likely a problem with the panoptes-front-end package,
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m not with npm itself.
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Tell the author that this fails on your system:
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m     export PATH_ROOT=preview.zooniverse.org/panoptes-front-end; npm run _build && npm run _publish && npm run _log-staging-url
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m You can get information on how to open an issue for this project with:
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m     npm bugs panoptes-front-end
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Or if that isn't available, you can get their info via:
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m     npm owner ls panoptes-front-end
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m There is likely additional logging output above.
[0m
[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Please include the following file with any support request:
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m     /src/npm-debug.log
[0m
[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Linux 3.13.0-123-generic
[0m[37;40mnpm[0m [0m[31;40mERR![0m [0m[35margv[0m "/usr/bin/nodejs" "/usr/bin/npm" "run" "stage"
[0m[37;40mnpm[0m [0m[31;40mERR![0m [0m[35mnode[0m v6.11.0
[0m[37;40mnpm[0m [0m[31;40mERR![0m [0m[35mnpm [0m v3.10.10
[0m[37;40mnpm[0m [0m[31;40mERR![0m [0m[35mcode[0m ELIFECYCLE
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m panoptes-front-end@ stage: `export NODE_ENV=staging; export SUBDIR=$(git symbolic-ref --short HEAD | awk '{print tolower($0)}'); npm run _stage`
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Exit status 1
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m 
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Failed at the panoptes-front-end@ stage script 'export NODE_ENV=staging; export SUBDIR=$(git symbolic-ref --short HEAD | awk '{print tolower($0)}'); npm run _stage'.
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Make sure you have the latest version of node.js and npm installed.
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m If you do, this is most likely a problem with the panoptes-front-end package,
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m not with npm itself.
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Tell the author that this fails on your system:
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m     export NODE_ENV=staging; export SUBDIR=$(git symbolic-ref --short HEAD | awk '{print tolower($0)}'); npm run _stage
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m You can get information on how to open an issue for this project with:
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m     npm bugs panoptes-front-end
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Or if that isn't available, you can get their info via:
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m     npm owner ls panoptes-front-end
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m There is likely additional logging output above.
[0m
[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m Please include the following file with any support request:
[0m[37;40mnpm[0m [0m[31;40mERR![0m[35m[0m     /src/npm-debug.log
```

--- a/webpack.build.js
+++ b/webpack.build.js
@@ -46,7 +46,7 @@ module.exports = {
     ]),
   ],
   resolve: {
-    extensions: ['.js', '.jsx', '.cjsx', '.coffee', '.styl', '.css'],
+    extensions: ['.js', '.jsx', '.json', '.cjsx', '.coffee', '.styl', '.css'],
     modules: ['.', 'node_modules'],
   },
   module: {

